tap_fixerio.py

```diff
@@ -100,8 +102,14 @@ def main():
 
     start_date = state.get('start_date',
                            config.get('start_date', datetime.utcnow().strftime(DATE_FORMAT)))
+    access_key = state.get('access_key', config.get('access_key'))
+
+    ## if access key is not supplied
+    if access_key is None:
+        logger.fatal("Please provide valid access key for Fixerio API")
+        sys.exit(-1)
```

Comment:
Assuming we already have some `class APIError(Exception)`, this is a great spot to also use it and we can even recycle the `"Please provide valid access key for Fixerio API"`.

Reply:
Where would I find if we have a class APIError implemented already?

Reply:
Oops, I was saying that assuming we made it to fix the change around line 63.

Reply:
This should also be raising an exception, like the change implemented in [this commit](https://github.com/singer-io/tap-fixerio/pull/5/commits/deef04d8e21b397cafeb9df63daf19a20ba872a3).

There are a few ways to get this behavior, but the most standard would be using the [singer-python argparser](https://github.com/singer-io/singer-python/blob/master/singer/utils.py#L126), with a list of required config keys. It will raise exceptions on its own when it parses. ([Example](https://github.com/singer-io/tap-zendesk/blob/master/tap_zendesk/__init__.py#L161))